packages/DraftEditor/src/LinkControl/LinkControl.tsx

```diff
@@ -0,0 +1,103 @@
+import Input from "@blaze-react/input";
+import Modal from "@blaze-react/modal";
+import { ContentState, EditorState, SelectionState } from "draft-js";
+import React, { useState } from "react";
+import StyleButton from "../StyleButton";
+
+import { IMMUTABLE, LINK } from "../constants";
+import { ILinkControlProps } from "../interfaces";
+
+const LinkControl = ({
+  editorState,
+  unSelectedText,
+  onToggle
+}: ILinkControlProps): JSX.Element => {
+  const [modalStatus, setModalStatus] = useState<boolean>(false);
+  const [url, setUrl] = useState<string>("");
+  const [selectedContent, setSelectedContent] = useState<SelectionState>();
+
+  const getSelection = (): void => {
+    const selection: SelectionState = editorState.getSelection();
+
+    const start: number = selection.getStartOffset();
+    const end: number = selection.getEndOffset();
+    if (start !== end) {
+      setSelectedContent(selection);
+    }
+  };
+
+  const openModal = (): void => {
+    getSelection();
+    toggleModal();
+  };
+
+  const addLink = (): void => {
+    const contentState: ContentState = editorState.getCurrentContent();
+
+    if (!selectedContent) {
+      return;
+    }
+
+    const contentStateWithEntity: ContentState = contentState.createEntity(
+      LINK,
+      IMMUTABLE,
+      { url }
+    );
+    const entityKey: string = contentStateWithEntity.getLastCreatedEntityKey();
+    const newEditorState: EditorState = EditorState.set(editorState, {
+      currentContent: contentStateWithEntity
+    });
+
+    onToggle(newEditorState, selectedContent, entityKey);
+    toggleModal();
+  };
+
+  const alertActions = [
+    {
+      callback: addLink,
+      modifiers: [
+        "small",
+        "rounded",
+        "outline",
+        `${!!selectedContent ? "" : "disabled"}`
```

Comment:
this ```!!``` is not necessary. selectedContent is inside ternary. Will be evaluated as a boolean, and if selectedContent is thruty will return "" if not will return disabled.